kalite/control_panel/tests/control_panel.py

```diff
@@ -551,60 +557,60 @@ def test_device_log_csv_endpoint(self):
         self.assertEqual(len(rows), 2, "API response incorrect")
 
 
-class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
+# class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
```

Comment:
I'm assuming you're removing these tests because they were giving too many false positives? Ideally, if we consider what they're _intended_ to test useful, then we should write new tests. Otherwise if they're not even useful in principle, then I'd just remove them instead of commenting them out.